### ISSUE TITLE
Allow Multiple Git Remotes

### ISSUE THREAD

**Issue by vHanda** [GitJournal/GitJournal#149]
Allow Repos to have more than 1 remote, and configure the sync frequency of each remote.

---
> Please vote on this issue with a 👍reaction. It's one of the primary mechanisms used to prioritize issues. Please also subscribe to the issue if you're interested, as that way I can discuss possible implementations and share mockups.

**Comment by tionis** [comment]
Shouldn't this be closed as finished?

**Comment by vHanda** [comment]
Nope. GitJournal allows multiple repos, but each repo only syncs with one remote. I can't have my repo in both GitHub and GitLab for example.
